al-sufi-arabic/description.md

```diff
@@ -0,0 +1,160 @@
+# Al-Şūfī Arabic
+
+## Introduction
+This sky culture reflects the indigenous star names, constellations and asterisms identified by the prominent astronomer: Al-Şūfī in his **Book of the Stars** (Arabic:كتاب الكواكب), written in the 10th century. The book maps the Arab names to the respective stars of the 48 classical Greek constellations in Ptolemy's Almagest.
+## Description
+**Book of the Stars** (Arabic:كتاب الكواكب) is an astronomical book written shortly before 964 AD by the astronomer Abu 'l-Ḥusayn 'Abd Al-Raḥmān ibn 'Umar Al-Şūfī (born in Rayy, 903; died 986), referred to as Al-Şūfī. It is the most outstanding among the works of the prominent astronomer, as Professor Paul Kunitzsch described it. In the book, Al-Şūfī treats the 48 classical constellations, which were established and transmitted in Ptolemy's Almagest. For each constellation, a chapter is dedicated that includes the following four sections: 
+1. A general discussion of the constellation and its individual stars; this section also includes al-Şūfi's criticism of the Ptolemaic tradition and notices of his own observations.
+2. A record of the indigenous Arabic star names falling under the constellation, and the exact identification of the respective stars with the corresponding Ptolemaic stars.
+3. Two drawings of the constellation, one as it is seen on a celestial globe, and the other one as it is seen in the sky.
+4. A table of the stars making up the constellation, giving a verbal description of each star's location and its longitude, latitude, and magnitude. This table closely follows the arrangement of Ptolemy's star catalogue in the Almagest."[^1].
+
+An important aspect of Al-Şūfī book is his record of the indigenous Arabic star names falling under each constellation, and the exact identification of the respective stars with the corresponding Ptolemaic stars. This means that we are looking at two layers of stars knowledge at the same time: the Ptolemaic one and the indigenous Arab one.
+
+The sky culture displays the indigenous ancient Arabs names of stars, constellations and asterisms. The lines in this sky culture represents the Arabs asterisms and constellations, where the Greek constellations are displayed with their figures and names without lines.
+## Constellations
+The sky culture contains the 48 Greek constellations listed by Ptolemy in the Almagest plus three constellations imagined by Al-Şūfī and were illustrated in his book.
```

Comment:
In the new format, the list is supposed to contain the description text for each constellations, if you only list the name it's enough to have this only in the index.json

Reply:
Done.